README.md

```diff
@@ -14,3 +16,38 @@ The initial use case for this SDK revolves around three main objectives:
   
 ### Export Annotations to Spark DataFrames/Delta Tables:
   The SDK should offer functionality to export annotations and store them in Spark DataFrames or Delta tables, facilitating further data analysis within Databricks.
+
+## Requirements:
+  * Databricks : Runtime 13.3 or later
+  * Clarifai : ` pip install clarifai `
+  * Create your [Clarifai account](https://clarifai.com/login)
+  * Protocol Buffers : version 4.24.2 `pip install protobuf==4.24.2 `
```

Comment:
Maybe in next release, Can we mention protobuf in requirements?!

Reply:
Yeah sure !.